_features/fresh.html

```diff
@@ -2,6 +2,14 @@
 title: Fresh
 slug: fresh
 description: Always up-to-date.
+usecases:
+- As a web application developer, I want to deliver updates seamlessly, without requiring
+  user intervention and approval.
+- As a user, I don't want to wait for the new version to be installed; I always
+  want the latest version available.
 technologies:
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
+<p>When users visit your web app, they are not prompted with a request to update,
+they simply see yout brand new UI with all your awesome functionality. As web apps, progressive web apps
+behave the same way out of the box but provide you with the means to fully customize the
+life cycle of your app.</p>
```

Comment:
> At the same time, I don't know if lifecycle-customizable (or whatever the adjective for that is) is actually a feature we want for PWA. Having said that, I would prefer to simply remove fresh from features.

Sounds good, I'm also unsure. There's something to be said for the ability to customize an app's lifecycle. But I don't know if it's valuable enough to describe it as a feature.

I do think there's huge value in being able to upgrade/downgrade an app reliably, ensuring that all resources required for a change have been downloaded and cached before committing the change. That's genuinely different from the traditional web (discounting AppCache), and it's a feature of all other app platforms, including web-based ones like Firefox OS, via packaging (or binary diffs) and installers that roll back installations in the event of failure. It might be worth calling this out as a feature such as "reliable upgrades/downgrades."